### ISSUE TITLE
Tillate flere valg i dropdown komponent

### ISSUE THREAD

**Issue by RonnyB71** [Altinn/app-frontend-react#54]
## Beskrivelse
Dagens dropdown komponent tillater kun ett valg. Denne må utvides til å kunne støtte flere valg.

## Vurderinger
Hvordan skal valgene vises og fjernes?
1. Inne i dropdown komponenten med en hake forann de valgte elementene, "velge" en gang til for å fjerne?
2. Som labels ved siden av dropdown listen med en 'x' for å fjerne?

Hvordan skal valge elementer lagres?

## Akseptansekriterier
- [ ] 

## Definition of done
- [ ] Documentation (docs.altinn.studio) is updated (if relevant)
  - [ ] Technical documentation
  - [ ] User documentation
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

**Comment by lorang92** [comment]
Duplikat av denne? https://github.com/Altinn/altinn-studio/issues/4933

**Comment by RonnyB71** [comment]
> Duplikat av denne? [Altinn/altinn-studio#4933](https://github.com/Altinn/altinn-studio/issues/4933)

Yep - og jeg forsøkte faktisk å lete :-D Lukker denne og beholder den andre.

**Comment by RonnyB71** [comment]
Duplikat av #4933
